cmd/server/cmd/server.go

```diff
@@ -140,6 +143,12 @@ func serverCmd(repo template.Repository, adapters []pkgAdapter.InfoFn, printf, f
 	serverCmd.PersistentFlags().StringVarP(&sa.configStoreURL, "configStoreURL", "", "",
 		"URL of the config store. May be fs:// for file system, or redis:// for redis url")
 
+	serverCmd.PersistentFlags().StringVarP(&sa.configStore2URL, "configStore2URL", "", "",
+		"URL of the config store. May be fs:// for file system, or redis:// for redis url")
+
+	serverCmd.PersistentFlags().StringVarP(&sa.configDefaultNamespace, "configDefaultNamespace", "", "default",
```

Comment:
I believe the default value shouldn't be `"default"` -- this is the namespace where many of k8s users are using by default.
Instead, use `"istio-config-default"` which is defined in https://github.com/istio/mixer/blob/master/pkg/runtime/resolver.go#L83.

Reply:
aah yes, good catch. changed it.